HW21/task2.py

```diff
@@ -0,0 +1,42 @@
+import requests
+
+def get_city_coordinates(city_name):
```

Comment:
nit: better to write docstring for functions or typings because it is not clear. For example, in this function, it would be nice to explain what data returns
```suggestion
def get_city_coordinates(city_name: str) -> tuple(str, str):
```
or

```suggestion
def get_city_coordinates(city_name):
    """Return tuple with city latitude and longitude"""
```